docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -137,7 +137,7 @@ in subsequent requests, just like browser cookies.
   * - Header name
     - Description
   * - ``X-Trino-User``
-    - Specifies the session user; must be supplied with every
+    - Specifies the session user; optional value to supply with every
```

Comment:
Let's remove the last part, since it's misleading and in the wrong place.

```
Specifies the session user. If not supplied, the session user will be
automatically determined via :doc:`/security/user-mapping`.

Reply:
Added with slight mods.